float32 irmse # Integrated Root Mean-Square Error
float32 imae
float32 mse # Mean square error
float32 rmse # Root mean square error
float32 mae # Mean absolute error
float32 absrel 
float32 lg10
float32 delta1
float32 delta2
float32 delta3
float32 margin10
float32 filtered
float32 data_time
float32 gpu_time
int64 count